app/ide-desktop/lib/client/src/config/parser.ts

```diff
@@ -316,35 +311,38 @@ export function parseArgs() {
     interface YargsArgs {
         // We don't control the naming of this third-party API.
         /* eslint-disable @typescript-eslint/naming-convention */
-        [key: string]: string[] | string | undefined
+        [key: string]: string[] | string
         _: string[]
         // Exists only when the `populate--` option is enabled.
         '--'?: string[]
         $0: string
         /* eslint-enable @typescript-eslint/naming-convention */
     }
 
-    let parseError: Error | undefined
+    // Required otherwise TypeScript thinks it's always `null`.
+    // eslint-disable-next-line no-restricted-syntax
+    let parseError = null as Error | null
```

Comment:
type assertion #4

Reply:
This is safe, right? Can we express it in safer way?